proxies/live/apiproxy/targets/target.xml

```diff
@@ -37,6 +23,42 @@
             </Step>
         </Request>
     </PreFlow>
+    <Flows>
+        <Flow name="App Restricted">
+            <Request>
```

Comment:
Could the condition be moved to the top of each flow so it's obvious that the flow is gated on first look